### DISCUSSION TITLE
Renovate Force Pushes Over Configuration Changes If You Amend Renovate's Own Commits

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#18628] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

On the advice of ["No risk onboarding"](https://docs.renovatebot.com/getting-started/installing-onboarding/#no-risk-onboarding), we edited the configuration pull requests Renovate automatically opened (e.g., ScribeMD/.github#94. Every time Renovate ran though, it [force pushed](https://github.com/ScribeMD/.github/compare/c4f2b7ce5bd14d35be8e0ce364f3b027c8d09d96..5f26368174abd93b87d64edca6e657c6bc3cb4b8#diff-1992243d5314f87446958f36bed759d287af9d80ef861d20051e8575cb208ffb) over any of our changes, completely resetting the branch to its original state, so we had to keep force pushing over those changes and then quickly merge before it had a chance to run. I assume this is because we were renaming `renovate.json` to `.github/renovate.json`, and Renovate was interpreting the removal of `renovate.json` as an instruction to ["Nuke config and re-onboard."](https://docs.renovatebot.com/getting-started/installing-onboarding/#nuke-config-and-re-onboard)

### Relevant debug logs

When I log into Renovate's app dashboard, it doesn't display any repos. I assume this is because we are using Forking Renovate.

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]
Note: Forking Renovate, it may be a special case.

Were you using your own git author/committer for the changes and adding additional commits to the branch, and not e.g. amending Renovate's own commit?

**Comment by Kurt-von-Laven** [comment, marked as answer]
I was [amending Renovates' own commit](https://github.com/ScribeMD/.github/pull/94/commits).

**Reply by rarkins** [reply]
Converted to a discussion as this is expected behavior. If you want to change a Renovate branch, do not use Renovate's identity to do so. e.g. add a commit using your own GitHub account.

**Reply by Kurt-von-Laven** [reply]
Thanks for the tip. That isn't compatible with our Git workflow, so we would simply have closed the auto-generated PRs if we had been aware of this. Is this already documented somewhere? Will come back through and open some documentation PRs once the dust settles.

**Reply by rarkins** [reply]
You're most welcome to submit docs improvement PRs once you're done. I'm not sure if we have stated that explicitly anywhere in user docs. I'll just say this though: there are many ways you can mess up Renovate if you mess with its git, and it's unlikely that we can ever document all of them.

BTW you can usually modify/force push Renovate's branches, but just don't do it while impersonating the bot in any way (such as amending the commit or explicitly committing with its git identity). So if your workflow is that you must have one commit per branch, it's still possible.
